### PR TITLE
fix background image size

### PR DESCRIPTION
**Pull request by szaimen**
| Before | After |
|---|---|
| image | image |

Signed-off-by: Simon L <szaimen@e.mail.de>

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -371,7 +371,7 @@ export default {
 		flex-shrink: 0;
 
 		.firstrunwizard-header {
-			background-size: var(--image-background-size, 275px, contain);
+			background-size: cover;
```
Comment (skjnldsv) [inline src/App.vue:374]: https://github.com/nextcloud/server/blob/8434259b1bfef22adbf30b166043e29763bf9507/apps/theming/lib/Themes/CommonThemeTrait.php#L111

We should drop the variable, it was here for the repeat transparent icon-like background we had in 24 and below.

**Review by skjnldsv — APPROVED** [review]

**Review by juliushaertl — APPROVED** [review]

File: src/App.vue (hunk 1 above)
Comment (szaimen) [reply src/App.vue:374]: done in https://github.com/nextcloud/server/pull/35547

**Pull request merged**